app/views/users/show.html.erb

```diff
@@ -1,2 +1,15 @@
-<h1>User Detail</h1>
-<h2>Posts</h2>
+<div class="container p-2">
+  <div class="row flex flex-column">
+    <%= render "shared/user_header", user: @user%>
+    <div class="bio border p-3 w-50">
+      <h4>Bio</h4>
+      <p> <%= @user.bio%> </p>
+    </div>
+    <div class="w-50 border p-3 mt-1">
+      <% @user.posts.each do |post| %>
```

Comment:
- [ ] The posts that should be rendered in this user show page are the recent posts of the user. Kindly refactor this view to use the `recent_three_posts` in your User model.

  ```
  @user.recent_three_posts.each do |post|
    ...
  ```